tests/e2e/scale/noobaa/test_scale_obc_creation_repsin_noobaa_pods.py

```diff
@@ -0,0 +1,118 @@
+import logging
+import pytest
+import time
+
+from ocs_ci.ocs import constants, scale_noobaa_lib
+from ocs_ci.framework.testlib import scale, E2ETest
+from ocs_ci.ocs.resources.objectconfigfile import ObjectConfFile
+from ocs_ci.framework.pytest_customization.marks import vsphere_platform_required
+
+log = logging.getLogger(__name__)
+
+
+@pytest.fixture(autouse=True)
+def teardown(request):
+    def finalizer():
+        scale_noobaa_lib.cleanup(constants.OPENSHIFT_STORAGE_NAMESPACE)
+
+    request.addfinalizer(finalizer)
+
+
+@scale
+class TestScaleOCBCreation(E2ETest):
+    """
+    OBC scale creation, creating up to max support number of OBCs.
+    OBCs are created in the Multicloud Object Gateway and
+    Ceph Object Gateway (RGW)
+
+    """
+
+    namespace = constants.OPENSHIFT_STORAGE_NAMESPACE
+    scale_obc_count = 1000
+    num_obc_batch = 50
+
+    @pytest.mark.parametrize(
+        argnames=["pod_name", "sc_name"],
+        argvalues=[
+            pytest.param(
+                *["noobaa-core", constants.NOOBAA_SC],
+                marks=[
+                    pytest.mark.polarion_id("OCS-2645"),
+                ],
+            ),
+            pytest.param(
+                *["noobaa-db", constants.NOOBAA_SC],
+                marks=[
+                    pytest.mark.polarion_id("OCS-2646"),
+                ],
+            ),
+            pytest.param(
+                *["noobaa-core", constants.DEFAULT_STORAGECLASS_RGW],
+                marks=[
+                    vsphere_platform_required,
+                    pytest.mark.polarion_id("OCS-2647"),
+                ],
+            ),
+            pytest.param(
+                *["noobaa-db", constants.DEFAULT_STORAGECLASS_RGW],
+                marks=[
+                    vsphere_platform_required,
```

Comment:
would this work for RHV or BM? or should we skip them?

Reply:
This should run on BM and RHV as well.  Changed to "on_prem_platform_required" instead.